src/components/Hjelpetekst.tsx

```diff
@@ -72,7 +78,11 @@ const Hjelpetekst: React.FC<Props> = ({
             lukkTekst={lukkeTekstid ? hentTekst(lukkeTekstid, intl) : undefined}
           >
             <Normaltekst>
-              <LocaleTekst tekst={innholdTekstid} />
+              {innholdTekst ? (
+                innholdTekst
+              ) : innholdTekstid ? (
+                <LocaleTekst tekst={innholdTekstid} />
+              ) : null}
```

Comment:
```suggestion
              {innholdTekst && (innholdTekst)}
              {!innholdTekst && innholdTekstId && <LocaleTekst tekst={innholdTekstid} />}
```

Bittelitt usikker på om ovennevnte funker, men da blir det iallefall _litt_ lettere for meg å se hva som skjer her. Andre (og kanskje mer lettleste) alternativer finnes helt sikkert :p